src/main/java/HW_1/Task1.java

```diff
@@ -0,0 +1,12 @@
+package HW_1;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+        String name = "Anastasiia";
+        int age = 30;
+        float weight = 53.5f;
+
+        System.out.println("name=" + name +" " + "age=" + age +" " + "weight=" + weight);}
```

Comment:
"}" symbol should be on the next line

Reply:
fixed

Reply:
not fixed 
![Screenshot_16](https://github.com/AntBelyi/ALevel_automation/assets/66726924/709315a5-3ad1-4eff-8432-f0ed829498e2)